appinfo/info.xml

```diff
@@ -5,6 +5,6 @@
     <description>A simple maps app</description>
     <licence>agpl</licence>
     <author>Sander Brand</author>
-    <version>0.0.6</version>
+    <version>0.0.6.1</version>
```

Comment:
I believe we should keep the [SemVer](http://semver.org/) format.
I understand that you increased it because of the database update, but we should either increase the patch (3rd) number (0.0.7) or leave it as is (0.0.6) because we haven't released it yet.